integration-tests/bats/doltpy.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+    dolt sql <<SQL
+CREATE TABLE foo (
+  a BIGINT NOT NULL,
+  b BIGINT,
+  PRIMARY KEY (a)
+);
+INSERT INTO foo VALUES (0,0), (1,1);
+SELECT DOLT_COMMIT('-am', 'Initialize table');
+SQL
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+# NOTE: if these break on a release that's OK, just tell Max/Oscar
+
+# already covered by other tests:
+# "select * from dolt_branches"
+# "SELECT * FROM `{table}` ASOF '{commit}'"
+
+# verifying the output comparison to dolt log would be nice
+@test "doltpy: Dolt.log" {
```

Comment:
I think you should put the assertion in the test name. Like:

"dolt log returns proper named headers"